setup.py

```diff
@@ -52,16 +52,19 @@
     # llfuse 1.2 (tested shortly, looks ok), needs FUSE version >= 2.8.0
     # llfuse 1.3 (tested shortly, looks ok), needs FUSE version >= 2.8.0
     # llfuse 2.0 will break API
-    'fuse': ['llfuse<2.0', ],
+    'fuse': [
+        'llfuse >=1.3.4, <1.3.7; python_version <"3.9"',  # broken on py39
+        'llfuse >=1.3.7, <2.0; python_version >="3.9"',  # broken on freebsd
+    ],
 }
 
 if sys.platform.startswith('freebsd'):
     # llfuse was frequently broken / did not build on freebsd
     # llfuse 0.41.1, 1.1 are ok
     extras_require['fuse'] = ['llfuse <2.0, !=0.42.*, !=0.43, !=1.0', ]
 
-if my_python >= (3, 7):
-    extras_require['fuse'][0] += ', >=1.3.4'
+# if my_python >= (3, 7):
+#     extras_require['fuse'][0] += ', >=1.3.4'
```

Comment:
good find! yeah, this did not match the code above any more.
guess the freebsd change right above also needs removal.

2 possibilities now:
- open a ticket about it and later fix master and 1.1-maint in a separate PR.
- have a bit of scope creep (this fix is not a backport, but a new change!) and remove (not: comment) the code here.

Reply:
mmm i think that master is ok ... your changes reflect what i changed in this branch

Reply:
right, master is ok, no such code there. so just remove it here.